src/e2e-test/resources/errorMessage.properties

```diff
@@ -13,3 +13,6 @@ errorMessagePubSubMaxBatchSizeField=Invalid maximum batch size. Ensure the value
 errorMessagePubSubDelayThresholdField=Invalid delay threshold for publishing a batch. Ensure the value is a positive number.
 errorMessagePubSubRetryTimeout=Invalid max retry timeout for retrying failed publish. Ensure the value is a positive number.
 errorMessagePubSubErrorThreshold=Invalid error threshold for publishing. Ensure the value is a positive number.
+errorMessageBQExecuteDatsetError=Dataset and table must be specified together.
```

Comment:
errorMessageBQExecuteTableDataset=Dataset and table must be specified together.

Reply:
@sawantpritam -updated